src/util/sss_krb5.c

```diff
@@ -973,12 +972,6 @@ krb5_error_code sss_extract_pac(krb5_context ctx,
         goto done;
     }
 
-    ret = unsetenv("_SSS_LOOPS");
-    if (ret != EOK) {
-        DEBUG(SSSDBG_CRIT_FAILURE, "Failed to unset _SSS_LOOPS, "
-                  "sss_pac_make_request will most certainly fail.\n");
-    }
-
```

Comment:
Why is this removed?

Reply:
Reintroducing the lines after the IRC discussion.